### PR TITLE
Reorganize readme

### PR DESCRIPTION
**Pull request by DavidBruant**

### CODE REVIEW

**Comment by DavidBruant** [comment]
I would like to request a review from @michaelfig, but i don't seem to have sufficient rights on the repo to do so

File: README.md [hunk 1]
```diff
@@ -1,41 +1,81 @@
 # Agoric's Cosmic SwingSet
 
-TL;DR: Request a public testnet provisioning code from https://testnet.agoric.com/, and
-after you have received the code run:
+Agoric's Cosmic SwingSet enables developers to test smart contracts build with [ERTP](https://github.com/Agoric/ERTP) in various blockchain setup environments
 
+This repository currently hosts various pieces:
```
Comment (DavidBruant) [inline README.md]: This is my current understanding and i'm pretty sure it's incomplete and/or inaccurate

File: README.md [hunk 2]
```diff
@@ -1,41 +1,81 @@
...
-$ SOLO_NAME=han ./docker/ag-setup-solo --pull
+
+Make symbolic links somewhere in your `$PATH` (such as `/usr/local/bin`) as below: 
+
+```sh
+ln -s $PWD/lib/ag-chain-cosmos /usr/local/bin/
 ```
 
-where `han` is the name of your solo vat machine that follows the blockchain.
+If installing the GO language didn't setup a `$GOPATH` variable,
```
Comment (DavidBruant) [inline README.md]: Under which circumstances would this not happen?

File: README.md [hunk 3]
```diff
@@ -1,41 +1,81 @@
 # Agoric's Cosmic SwingSet
 
-TL;DR: Request a public testnet provisioning code from https://testnet.agoric.com/, and
-after you have received the code run:
+Agoric's Cosmic SwingSet enables developers to test smart contracts build with [ERTP](https://github.com/Agoric/ERTP) in various blockchain setup environments
 
+This repository currently hosts various pieces:
+- Code to set up a "solo node" (a client program which purpose is to discuss with a blockchain on a testnet)
```
Comment (DavidBruant) [inline README.md]: I'm not sure on the definition of this one

File: README.md [hunk 4]
```diff
@@ -1,41 +1,81 @@
...
-  $ make scenario3-run-client
-  ```
+Run:
+```sh
+make scenario3-setup
+make scenario3-run-client
+```
 
-  The `lib/ag-solo/vats/` directory contains the source code for all the Vats
+The `lib/ag-solo/vats/` directory contains the source code for all the Vats
```
Comment (DavidBruant) [inline README.md]: @michaelfig i have in mind what you said about this part being out of date. I'm not fully sure to understand what the relevant replacement would be yet, but i'll work on it

File: README.md [hunk 5]
```diff
@@ -234,6 +256,25 @@ ag-setup-cosmos bootstrap --bump
 ag-setup-cosmos destroy
 ```
 
+## With a provisioning code from Agoric
+
+You can request a public testnet provisioning code from https://testnet.agoric.com/
```
Comment (DavidBruant) [inline README.md]: How can people request a provisionning code? Who should they write to?

File: README.md [hunk 6]
```diff
@@ -234,6 +256,25 @@ ag-setup-cosmos bootstrap --bump
...
+
+Once you have received the code run:
+
+```
+$ SOLO_NAME=han ./docker/ag-setup-solo --pull
+```
+
+where `han` is the name of your solo vat machine that follows the blockchain.
```
Comment (DavidBruant) [inline README.md]: The command does not say how the provisionning code is "inserted" into the mix.
Is it a file people place somewhere? Is it an environment variable?

File: README.md (hunk 5 above)
Comment (michaelfig) [reply README.md]: They visit the linked website.  That's where they request the code.

File: README.md (hunk 6 above)
Comment (michaelfig) [reply README.md]: It is a code that they paste or type when prompted by ag-setup-solo.

File: README.md (hunk 3 above)
Comment (michaelfig) [reply README.md]: I'd change to:
> (a local peer which can host objects for testing or interaction with a blockchain)

File: README.md (hunk 2 above)
Comment (michaelfig) [reply README.md]: It happened to @erights, and he submitted the doc change, but I'm not sure what caused it.

File: README.md (hunk 4 above)
Comment (michaelfig) [reply README.md]: Maybe just a pointer to `lib/ag-solo/vats/vat-demo.js` as a starting point.  You might want to mention that this `vats` directory is copied into the directory named by `ag-solo init DIRECTORY`.

File: README.md (hunk 5 above)
Comment (DavidBruant) [reply README.md]: oh yes, sorry, i had misunderstood the purpose of the link

File: README.md (hunk 6 above)
Comment (DavidBruant) [reply README.md]: ok cool, i'll add a note about this
Thanks!

File: README.md (hunk 2 above)
Comment (DavidBruant) [reply README.md]: Ok
I'm leaving the note and the comment
I think this is not a blocking concern for this PR

File: README.md [hunk 7]
```diff
@@ -75,14 +75,13 @@ make scenario3-setup
...
-created in the solo vat-machine. The actual filenames are enumerated in
-`lib/ag-solo/init-basedir.js`, so if you add a new Vat, be sure to add it to
-`init-basedir.js` too.
+[`lib/ag-solo/vats/vat-demo.js`](lib/ag-solo/vats/vat-demo.js) contains the code running a vat with the Pixel Gallery Demo.
 
-The objects added to `home` are created in `lib/ag-solo/vats/bootstrap.js`.
+Also, as part of `make scenario3-setup`, `bin/ag-solo init <directory>` get called and all the content of the [`vats`](lib/ag-solo/vats) directory gets copied to the `<directory>`
 
-The REPL handler is in `lib/ag-solo/vats/vat-http.js`.
+The objects added to `home` are created in [`lib/ag-solo/vats/bootstrap.js`](lib/ag-solo/vats/bootstrap.js).
```
Comment (michaelfig) [inline README.md]: They're actually created in `lib/ag-solo/vats/vat-demo.js`.

**Pull request merged**